### DISCUSSION TITLE
可否保留之前版本的页面模版？

### DISCUSSION THREAD

**Discussion by 52eraja** [usememos/memos#1307] [Ideas]
新版本的模版是更清晰布局合理，但是我现在在折叠屏手机使用情况下，页面出现了叠加和错位，十分影响使用体验。所以是否可以保留旧版本的页面风格或者用户可以自行选择。谢谢。![Screenshot_2023-03-08-13-11-59-83_8b7aa5be40a89c6a7df806c347da652a.jpg](https://user-images.githubusercontent.com/122664422/223625890-825fcca4-29af-488e-8443-884c83d5700e.jpg)

![Screenshot_2023-03-08-13-05-59-03_df198e732186825c8df26e3c5a10d7cd.jpg](https://user-images.githubusercontent.com/122664422/223625898-54480a8e-81cd-4de5-bf8c-669eb316c1e1.jpg)

**Comment by wer1030** [comment]
在手机上基本上不能用了，都点不到左边的目录和展开。
